react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx

```diff
@@ -90,13 +90,28 @@ const ModalView: FC<{ closeAction: () => void }> = ({
 
 interface Props {
   buttonProps?: ButtonProps
-  modalProps?: unknown
+  modalProps?: any
 }
 
 interface ButtonProps {
   collapse?: 'left' | 'right' | 'none'
 }
 
+const hasValues = (itemChildren: any) => {
+  const [name] = Object.getOwnPropertyNames(itemChildren)
+  const inputs = itemChildren[name].inputValues
+
+  if (!itemChildren[name]?.valuesOfInputValues) {
```

Comment:
Here you conditionally access the property (?.) and on line 102 you don't. Is there a reason for it?

Reply:
Good point, changing it.